src/components/LocationSearch/styles.js

```diff
@@ -0,0 +1,90 @@
+import styled, { css } from "styled-components";
+
+export const LocationListWrapper = styled.div`
+  max-height: 90%;
+  overflow: auto;
+  padding-right: 1rem;
+  margin-top: var(--padding-size-medium);
+`;
+
+export const LocationSearchWrapper = styled.div`
+  width: 100%;
+  height: 100%;
+  position: absolute;
+  padding-top: var(--font-size-medium);
+  top: 0;
+  left: 0;
+  z-index: 100;
+  padding: var(--padding-size-medium);
+  background-color: var(--color-primary);
+  transform: translateX(-100%);
+  transition: all ease-in-out 0.4s;
+
+  ${(props) =>
+    props.$isLocationSearchOpen &&
```

Comment:
You can use destructuring

```suggestion
  ${({ isLocationSearchOpen }) =>
    isLocationSearchOpen &&
```